README.md

```diff
@@ -128,9 +128,9 @@ We also have a generator for rails scaffold CSS. Just like the Sass gem, we simp
 This gem is fully tested from Rails 4.0 to Rails 4.2. We run our tests on [Travis CI](http://travis-ci.org/metaskills/less-rails) in both Ruby 1.9, 2.0, and jRuby 1.9 mode. If you detect a problem, open up a github issue or fork the repo and help out. After you fork or clone the repository, the following commands will get you up and running on the test suite. 
 
 ```shell
-$ bundle install
-$ bundle exec rake appraisal:setup
-$ bundle exec rake appraisal test
+$ bundle
```

Comment:
I can do that, but `bundle` is the same as `bundle install` (install is the default action). Let me know if you still want it changed back.